src/components/dropdown-select/dropdown-select.tsx

```diff
@@ -0,0 +1,38 @@
+import {Component, h, Prop} from "@stencil/core";
```

Comment:
lets name it `rg-attribute-dropdown-select` due to the fact that this component is specificly suited for that place